### PR TITLE
`General`: Fix alert layout

### LINKED ISSUE #4524: Client Alert Styling Partially Broken
### Describe the bug

The styling of the close button of the alert now has a button around it. Before, the button was not visible and just the black x was shown without a special background.

### To Reproduce

1. Trigger any kind of alert in the client, e.g. by changing a programming exercise.

### Expected behavior

The close icon is usually to the right without the special button-box around it.

### Screenshots

Screenshot 2021-12-18 at 13-14-35 Exercise Assessment Dashboard

### What browsers are you seeing the problem on?

Firefox 95.0.1

### Additional context

On current `develop` branch.

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by willergeorg**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context

This PR fixes #4524 

### Description

The layout for the alerts are currently broken as the close button is in the next line
### Steps for Testing

Testing this is quite tricky. The non plus ultra would be to test every possible alert on the client side and check whether the layout is still good.
Some of them appear when:
- Creating an exercise
- Submitting to an exercise
- Creating an exam
- ...?

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Test Coverage

Nothing should have changed
### Screenshots

The current layout:
image

How it should look now:
image

### CODE REVIEW

File: src/main/webapp/content/scss/global.scss [hunk 1]
```diff
@@ -313,6 +313,8 @@ Custom alerts for notification
 .alerts {
     .alert {
         text-overflow: ellipsis;
+        display: flex;
+        justify-content: space-between;
```
Comment (b-fein) [inline src/main/webapp/content/scss/global.scss:317]:
```suggestion
        justify-content: space-between;

        .close {
            background: none;
            border: none;
        }
```
Additional rule to fix the button styling.

**Review by b-fein — APPROVED** [review]
Tested locally: Both the actual alerts and the code look good. :+1:

**Review by Malyuk-A — APPROVED** [review]
Code changes are minimal and seem reasonable (and harmless).
Tested on TS1. The "buggy" look of the alerts is now fixed

**Review by Hialus — APPROVED** [review]
Tested locally and looks good now.
The CSS changes also look reasonable.
Approve 👍🏻

**Review by Strohgelaender — APPROVED** [review]
Tested on TS1, alerts look good again. 👍

**Review by mallgaier — APPROVED** [review]
Tested on ts1.
The alert notifications work as expected again.

**Review by filip-gregurevic — APPROVED** [review]

**Pull request merged**